x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema_logic.test.ts

```diff
@@ -468,7 +535,12 @@ describe('SchemaLogic', () => {
         SchemaLogic.actions.onFieldUpdate({ schema: newSchema, formUnchanged: false });
         SchemaLogic.actions.setFilterValue('foo');
 
-        expect(SchemaLogic.values.filteredSchemaFields).toEqual(schema);
+        expect(SchemaLogic.values).toEqual({
+          ...setupExpectedState({ didInitialize: true, setSchema: newSchema }),
```

Comment:
As stated above, this tries to fix the issue readability of the test. We are not interested on field update logic, but `filteredSchemaValues` here. Setting everything in the test doesn't help on readability either but it is easier to maintain..